### PR TITLE
Fixed data register on firestore

### PR DESCRIPTION
**Pull request by ljau**
Registers properly new users and new added flights

Co-authored-by: 96vladi <maidana.vladi@gmail.com>

### CODE REVIEW

File: src/containers/BookingReceived/index.js [hunk 1]
```diff
@@ -13,11 +13,17 @@ import auth from '@react-native-firebase/auth';
 export const Received = ({route: {params}, navigation}) => {
 
   const addFlights = () => {
+
+
+    console.log("THIS IS THE USER", auth().currentUser.uid)
+
```
Comment (DemisRincon) [inline src/containers/BookingReceived/index.js]: remove console logs

File: src/containers/BookingReceived/index.js [hunk 2]
```diff
@@ -13,11 +13,17 @@ import auth from '@react-native-firebase/auth';
...
     firestore()
       .collection('reservas')
       .doc(auth().currentUser.uid)
       .get()
       .then(response => {
+
+      console.log("THIS IS THE RESPONSE", response.data())
```
Comment (DemisRincon) [inline src/containers/BookingReceived/index.js]: remove console logs

File: src/library/methods/auth.js [hunk 3]
```diff
@@ -1,18 +1,20 @@
 import auth from '@react-native-firebase/auth';
 import { GoogleSignin } from '@react-native-google-signin/google-signin';
+import firestore from '@react-native-firebase/firestore';
 
 export const register = async (email, password, username) => {
   return await auth()
     .createUserWithEmailAndPassword(email, password)
     .then(({user}) => {
+      console.log("THIS IS THE USER", auth().currentUser)
+
```
Comment (DemisRincon) [inline src/library/methods/auth.js]: remove console log

**Review by DemisRincon — CHANGES_REQUESTED** [review]
check the comments in general good

File: src/containers/BookingReceived/index.js (hunk 1 above)
Comment (ljau) [reply src/containers/BookingReceived/index.js]: logs removed

File: src/containers/BookingReceived/index.js (hunk 2 above)
Comment (ljau) [reply src/containers/BookingReceived/index.js]: logs removed

File: src/library/methods/auth.js (hunk 3 above)
Comment (ljau) [reply src/library/methods/auth.js]: logs removed

**Review by RaulMorales404 — APPROVED** [review]

**Review by Shadic78 — APPROVED** [review]
Good

**Review by DemisRincon — APPROVED** [review]

**Pull request merged**